ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/PartyManagementServiceIT.scala

```diff
@@ -449,89 +446,4 @@ final class PartyManagementServiceIT extends PartyManagementITBase {
     }
   })
 
-  test(
-    "UpdateAnnotationsOfNonLocalParty",
-    "Update annotations of a non-local party",
```

Comment:
Not sure if already in this PR, but we probably need a conformance test that fetches a party that is not in the current idp but is local in another

Reply:
This will be covered with comprehensive IDP conformance tests PR